Disable bandit-baseline command. It depends on GitPython moduel that is not
part of Solaris now.

diff -ruN bandit-1.6.2.orig/setup.cfg bandit-1.6.2/setup.cfg
--- bandit-1.6.2.orig/setup.cfg	2020-07-02 16:23:57.973003692 +0000
+++ bandit-1.6.2/setup.cfg	2020-07-02 16:29:40.844380552 +0000
@@ -28,7 +28,6 @@
 console_scripts = 
 	bandit = bandit.cli.main:main
 	bandit-config-generator = bandit.cli.config_generator:main
-	bandit-baseline = bandit.cli.baseline:main
 bandit.blacklists = 
 	calls = bandit.blacklists.calls:gen_blacklist
 	imports = bandit.blacklists.imports:gen_blacklist
